app/charts/map/map-tooltip.tsx

```diff
@@ -183,3 +152,36 @@ export const MapTooltip = () => {
     </>
   );
 };
+
+interface TooltipRowProps {
+  title: string;
+  background: string;
+  color: string;
+  value: string;
+  error: string | null;
+}
```

Comment:
Nit: Since there is no particular advantage from using `interface` over `type`, and we have a bit more type for props, let's settle on using `type`.

Reply:
Ok, I will change that :)